src/components/tests/Header.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Header from '../Header';
+
+describe('selector types', () => {
+  it('select the search and test if the icon is rendered', () => {
```

Comment:
Preferably use `test()` with jest.
I do not see how you assure the icon is rendered, the test describes there's a list with a single element.